api/proxy/cmd/server/entrypoint.go

```diff
@@ -76,26 +76,24 @@ func StartProxyService(cliCtx *cli.Context) error {
 	}
 
 	restEnabledCfg := cfg.EnabledServersConfig.RestAPIConfig
-	if restEnabledCfg.Enabled() {
-		if err := restServer.Start(router); err != nil {
-			return fmt.Errorf("start proxy rest server: %w", err)
-		}
-
-		log.Info("Started EigenDA Proxy REST ALT DA server",
-			string(enabled_apis.Admin), restEnabledCfg.Admin,
-			string(enabled_apis.StandardCommitment), restEnabledCfg.StandardCommitment,
-			string(enabled_apis.OpGenericCommitment), restEnabledCfg.OpGenericCommitment,
-			string(enabled_apis.OpKeccakCommitment), restEnabledCfg.OpKeccakCommitment)
+	if err := restServer.Start(router); err != nil {
+		return fmt.Errorf("start proxy rest server: %w", err)
+	}
```

Comment:
you'll prob want to change https://github.com/Layr-Labs/eigenda/blob/364bb2dbd49f1b43a16dcc0e7c959a016f8eff81/api/proxy/config/enablement/enabled_apis.go#L29 to say CertEndpointEnabled() or something. And comment that config endpoint is always enabled by default.